src/docfx/lib/github/GitHubUserCache.cs

```diff
@@ -248,7 +248,7 @@ async Task<(Error error, bool collide)> SaveRemote(string content)
         /// 2. User missing with the specified login: { "login": "..." }
         /// 3. User missing with the specified email: { "emails": [ "..." ] }
         /// </summary>
-        private void UpdateUser(GitHubUser user)
+        private void UpdateUser(GitHubUser user, bool preferExistingName = false)
```

Comment:
when will it be false?

Reply:
All other places where users are updated:
- Users returned from _get by login` API
- Users read from cache file itself